yaks/lib/yaks/default_policy.rb

```diff
@@ -5,7 +5,7 @@ class DefaultPolicy
     # Default policy options.
     DEFAULTS = {
       rel_template: "rel:{rel}",
-      namespace: Kernel
+      namespace: Object
```

Comment:
Got it! `Kernel.const_get("A", false)` does not work, only `Object.const_get("A", false)` does. You may easily try it out with irb.